src/components/Theme/constants.scss

```diff
@@ -51,7 +51,7 @@ $color-black: rgb(var(--ods-color-neutral-black));
 
 $color-icon-temporary: rgba(var(--ods-color-neutral-300));
 $color-divider: rgba(var(--ods-color-neutral-400));
-$color-icons-connector-line: rgba(var(--ods-color-neutral-800));
+$color-icons-connector-line: rgb(var(--ods-color-neutral-800));
```

Comment:
This was because it wasn't showing up in IE11. Seems this colour value doesn't have an alpha value which is handled by other browsers, but is invalid for IE11. 

I'll revert it for this PR and handle this & a couple other IE11 specific CSS issues in a separate PR.